### PR TITLE
create and manage `unicore.hub` app credentials

### PR DESCRIPTION
**Pull request by Rizziepit**

### CODE REVIEW

**Comment by Rizziepit** [comment]
@miltontony ready for review. It needs a new release of https://github.com/universalcore/unicore.hub.client though.

**Comment by Rizziepit** [comment]
@miltontony this PR is passing now. Review is not urgent.

File: unicoremc/managers/settings.py [hunk 1]
```diff
@@ -72,6 +72,13 @@ def write_frontend_settings(
         for lang in available_languages:
             languages.append(repr((lang.get_code(), lang.get_display_name())))
 
+        if hub_app:
+            hub_app_id = hub_app.get('uuid')
```
Comment (miltontony) [inline unicoremc/managers/settings.py]: or perhaps this instead of the _if block_ `hub_app_id = hub_app.get('uuid') if hub_app else None`

**Comment by miltontony** [comment]
:+1:

**Pull request merged**